Improving documentation related to Live Message feature (#3137)
tracim/tracim: README.md

You can add "-d" to build_full_frontend.sh to disable obfuscation and reduce build time.

> ### Install The Pushpin-Proxy for Live Messages

Reviewer: ```suggestion
### Install and run pushpin for UI updates
```

Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.

> expected.

Reviewer: I think we can remove this line.

Author: ```suggestion
```

> Tracim rely on pushpin for live_message feature, so you need to have a working pushpin in order to make tracim work as

Reviewer: ```suggestion
Tracim relies on [pushpin](https://pushpin.org/docs/about/) for UI updates, so you need to setup pushpin.
```

## For development purpose

If you are in development environnement, the recommended way is to use docker,
you should follow this documentation : [Live message setup for dev env](/backend/doc/live_message_setup.md)

## For production usage

You should install pushpin, for example in debian/ubuntu:

>     apt install pushpin

Reviewer 2: ```suggestion
    sudo apt install pushpin
```

and configure it to proxy tracim (by default tracim web run on port 6543):

>     echo "* localhost:6543" > /etc/pushpin/routes

Reviewer 2: ```suggestion
    echo "* localhost:6543" | sudo tee /etc/pushpin/routes
```

and after this change you need to restart your pushpin service:

>     systemctl restart pushpin

Reviewer 2: ```suggestion
    sudo systemctl restart pushpin
```

    # email fetcher (if email reply is enabled)
    python3 daemons/mail_fetcher.py &

You can now head to (if pushpin is correctly configured and use default port 7999)
[http://127.0.0.1:7999](http://127.0.0.1:7999) and login with the admin account:

 * user: `admin@admin.admin`
 * password: `admin@admin.admin`

> :warning: If this does not work, you can try first:
> [http://127.0.0.1:6543](http://127.0.0.1:6543) and if it's working.
> It's a pushpin configuration issue.

Reviewer 2: ```suggestion
:warning: If this does not work, you can try to access [http://127.0.0.1:6543](http://127.0.0.1:6543). If it works, the issue is related to the configuration of pushpin.
```
